OADP-3215: Implement suggestions related to MCG (#89042)
openshift/openshift-docs: modules/migration-configuring-mcg.adoc

.Procedure
ifdef::installing-3-4,installing-mtc[]
* Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, and the `AWS_SECRET_ACCESS_KEY` value by running the `oc describe` command for the `NooBaa` CR.
+
You use these credentials to add MCG as a replication repository.
endif::[]
ifdef::installing-oadp-mcg[]

> . Create a MCG bucket. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.18/html-single/managing_hybrid_and_multicloud_resources/index[Managing hybrid and multicloud resources].

Reviewer: This is confusing, but here we'd use "an" instead of "a" because the "M" in "MCG" is vowel-sending (as in "empty"). See https://www.ibm.com/docs/en/ibm-style?topic=grammar-articles for more information if warranted. 

```suggestion
. Create an MCG bucket. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.18/html-single/managing_hybrid_and_multicloud_resources/index[Managing hybrid and multicloud resources].
```

You use these credentials to add MCG as a replication repository.
endif::[]
ifdef::installing-oadp-mcg[]

> . Create a MCG bucket. For details, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.17/html-single/managing_hybrid_and_multicloud_resources/index[Managing hybrid and multicloud resources].

Reviewer 2: ```suggestion
. Create a MCG bucket. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.17/html-single/managing_hybrid_and_multicloud_resources/index[Managing hybrid and multicloud resources].
```

Should the link be to 4.17 for all versions of this page?

Author: For now, I've updated it to 4.18 but I'll have to check it with the ODF team if they have any attribute for the release version that can be used in such cases.

> . Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and bucket name by running the link:https://docs.redhat.com/en/documentation//red_hat_openshift_data_foundation/4.15/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] on the bucket resource.

Reviewer 2: `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` should be functioning as adjectives, e.g. "`AWS_ACCESS_KEY_ID` value" or something.

Reviewer 2: Is linking to the command important here?

> . Obtain the S3 endpoint, `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and bucket name by running the link:https://docs.redhat.com/en/documentation//red_hat_openshift_data_foundation/4.15/html/managing_hybrid_and_multicloud_resources/accessing-the-multicloud-object-gateway-with-your-applications_rhodf#accessing-the-Multicloud-object-gateway-from-the-terminal_rhodf[`describe` command] on the created bucket resource.

Author: ack, done

EOF
----
+

> You can use the `credentials-velero` file to create a `Secret` object when you install the Data Protection Application. 

Reviewer 2: ```suggestion
You can use the `credentials-velero` file to create a `Secret` object when you install the Data Protection Application.
```
